tools/train.py

```diff
@@ -114,6 +114,9 @@ def main():
     else:
         distributed = True
         init_dist(args.launcher, **cfg.dist_params)
+        # re-set gpu_ids with distributed training mode
+        _, world_size = get_dist_info()
```

Comment:
Should they be put in line 109?

Reply:
The args.gpus defaultly is set to None when distributed training mode.